src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -133,19 +146,34 @@ export class SchoolSelectField extends React.Component {
         <div>
           <div className="search-controls">
             <Element name="schoolSearch"/>
-            <input
-              onChange={this.handleSearchInputChange}
-              onKeyDown={this.handleSearchInputKeyDown}
-              type="text"
-              value={searchInputValue}/>
-            <button
-              className="search-schools-button usa-button-primary"
-              onClick={this.handleSearchClick}>
-              {'Search Schools'}
-            </button>
+            <div className="search-input">
+              <input
+                onChange={this.handleSearchInputChange}
+                onKeyDown={this.handleSearchInputKeyDown}
+                type="text"
+                value={searchInputValue}/>
+              <div
+                className={clearSearchInfoClass}>
+                {showInstitutions && <span>
+                  {`${searchResultsCount} results for ${institutionQuery}`}
+                </span>}
+                <button
+                  onClick={this.handleStartOver}
+                  className="va-button-link">
```

Comment:
Did some reordering to fit the new flexbox layout. This button is the only new component.